hack/Makefile.debug

```diff
@@ -3,14 +3,21 @@
 export GOOS=linux
 
 REGISTRY ?= quay.io
-IMAGE ?= openshift/openshift-router
+IMAGE ?= $(USER)/openshift-router
+IMAGE_BASE ?= $(USER)/openshift-router-base
 TAG ?= latest
 IMAGEBUILDER ?= podman
 
 new-openshift-router-image:
 	GO111MODULE=on CGO_ENABLED=0 GOFLAGS=-mod=vendor go build -o openshift-router -gcflags=all="-N -l" ./cmd/openshift-router
 	$(IMAGEBUILDER) build -t $(IMAGE):$(TAG) -f hack/Dockerfile.debug .
 
+new-ci-router-image:
+	$(IMAGEBUILDER) build -t $(IMAGE_BASE):$(TAG) -f images/router/base/Dockerfile.rhel .
+	cp -f images/router/haproxy/Dockerfile.rhel8 images/router/haproxy/Dockerfile.rhel8.debug
+	sed -i "s|FROM .*|FROM $(IMAGE_BASE):$(TAG)|" images/router/haproxy/Dockerfile.rhel8.debug
```

Comment:
Can we use dockerfile build args https://docs.docker.com/build/guide/build-args/? You can apply default arguments.

Reply:
The rebase conflict I'm having on this PR is due to [the ART automation](https://github.com/openshift/router/commit/4b3e2655a7590e21dfdaab7c86c3ae5365842a4c) which updates `images/router/haproxy/Dockerfile.rhel8`'s `FROM`. The `ARG` may be preventing this automation from working smoothly.